main.js

```diff
@@ -0,0 +1,19 @@
+const openMenu = document.querySelector('.hamburger');
+const closeMenu = document.querySelector('.close-menu');
+const mobileMenu = document.querySelector('.mobile-menu ');
+
+openMenu.addEventListener('click', () => {
```

Comment:
- [ ] The "contact" link still does not go to the correct section, as the last reviewer requested.  Please, fix this link so it goes to the correct section.

## Hints

- The link does not work because it refers to a "#Contacts" section id that does not exist.
- Just add an id to this section and name it "Contacts".
![image](https://user-images.githubusercontent.com/57421823/163131697-1f98bbe7-8765-43b3-bef4-afb3b3cd27e1.png)

Like this:

```HTML
<section class="details" id="Contacts">
```

That way, the link will work! (Tested on Firefox). 💪